detekt/rules/src/main/kotlin/aditogether/detekt/rules/AdiTogetherRuleSetProvider.kt

```diff
@@ -0,0 +1,15 @@
+package aditogether.detekt.rules
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.RuleSet
+import io.gitlab.arturbosch.detekt.api.RuleSetProvider
+
+class AdiTogetherRuleSetProvider : RuleSetProvider {
+
```

Comment:
Pointless feedback: this line triggers my OCD, but removing that probably triggers yours 😂

Reply:
Exactly 😁 I like giving some air to the code. Let's take a decision when we'll introduce the code style